### PR TITLE
Simple tidy ups

### PR DESCRIPTION
**Pull request by ltratt**
This PR contains a few manually determined tidy ups: each commit has a bit more detail.

### CODE REVIEW

**Comment by vext01** [comment]
No one is assigned to this. Shall I do it?

**Comment by ltratt** [comment]
Oops, good idea!

File: internal_ws/ykcompile/src/lib.rs [hunk 1]
```diff
@@ -130,26 +129,6 @@ pub struct CompiledTrace {
 }
 
 impl CompiledTrace {
-    /// Execute the trace by calling (not jumping to) the first instruction's address. Returns a
```
Comment (vext01) [inline internal_ws/ykcompile/src/lib.rs:133]: I'm very surprised by the removal of this hunk. Investigating.

Comment (vext01) [reply internal_ws/ykcompile/src/lib.rs:133]: I see, this code was copied into `ykshim_client` and the old version wasn't removed.

**Comment by vext01** [comment]
Ready to merge?

**Comment by ltratt** [comment]
Go ahead!

**Comment by ltratt** [comment]
rustfmt problem fixed in https://github.com/softdevteam/yk/pull/308/commits/0e5a3452d7e46554f59cf457c83c9393b27c57ce.

**Comment by ltratt** [comment]
OK to squash?

**Comment by vext01** [comment]
Please go ahead.

**Comment by ltratt** [comment]
Squashed.

**Comment by vext01** [comment]
this has gotten stuck by the look of it.

bors r-

**Pull request merged**